code.ts

```diff
@@ -20,26 +20,29 @@ ref.forEach((layer:any) => {
 
   // make sure it's a vector
   if (layer.type === "RECTANGLE" || layer.type === "ELLIPSE" || layer.type === "POLYGON" || layer.type === "VECTOR") {
+    var fill = layer.fills[0]
     
-    if(!layer.fillStyleId){
+    if (!fill) {
+      // Just skip.
+    } else if (!layer.fillStyleId) {
```

Comment:
Wouldn't this line be more succinct with combining it with the rule below? Like:

```suggestion
    if (!fill && !layer.fillStyleId) {
```

Reply:
Also, this skip line doesn't trigger the error at the end of the if statement

Reply:
Actually, if you just add this line it will show the notification:

```suggestion
    if (!fill) {
      // Just skip.
      ignoredCounter++;
    } else if (!layer.fillStyleId) {
```